tests/outputs/u_milestone/projecta.yaml

```diff
@@ -187,7 +187,7 @@ sb_firstprojectum:
   deliverable:
     audience:
       - beginning grad in chemistry
-    due_date: 2020-06-01
+    due_date: '2021-05-05'
```

Comment:
you are changing the test outputs here.   This will have a lot of knock on effects.  That you have to change it maybe suggests something needs to be changed elsewhere?  We can discuss.